src/exper.py

```diff
@@ -47,18 +50,22 @@
     wandb.config.update(opts.to_dict())
     wandb.config.update({"__message": parsed_opts.message})
 
+    # load normalization
```

Comment:
Is this data on the cluster somewhere? Do you have the script you used to compute it?

Reply:
Okay, I see the normalization script below. I wonder if you could read the data into a transform method, though, so you could just pass in a list of transforms, rather than having lots of checks.

Reply:
I'm not sure I got your comment. Do you mean save the transform object? The idea is that we'd have pre-computed mean and std for all channels + slope + snow_index and it can be indexed later based on the the values in the config file